types/serialization.go

```diff
@@ -307,6 +307,9 @@ func (s *State) FromProto(other *pb.State) error {
 }
 
 func txsToByteSlices(txs Txs) [][]byte {
+	if txs == nil {
+		return nil
+	}
```

Comment:
specifically, is this captured in a test somewhere? What is the impact of not having this?

Reply:
Captured in `serialization_test.go`